mongoexport/options_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) MongoDB, Inc. 2014-present.
```

Comment:
There will be more tests here to test specifying a read preference in the URI once we figure out how to do TOOLS-2217.

Reply:
In the meantime, you can add an integration test for read preference behavior if you want. Given the way common is structured, it should still be getting set on the client even if it's absent from `ToolOptions`.

Reply:
I'm starting 2217 now so I'll add the tests as soon as that's done.